src/AutosuggestContainer.js

```diff
@@ -69,13 +69,14 @@ export default class AutosuggestContainer extends Component {
     renderSectionTitle: PropTypes.func,
     getSectionSuggestions: PropTypes.func,
     focusInputOnSuggestionClick: PropTypes.bool,
+    selectFirstSuggestion: PropTypes.bool,
     theme: PropTypes.object,
     id: PropTypes.string
   };
 
   static defaultProps = {
     onSuggestionsUpdateRequested: noop,
-    shouldRenderSuggestions: value => value.trim().length > 0,
+    shouldRenderSuggestions: value => value && value.trim().length > 0,
```

Comment:
> Default props in the library? Where is that?

`valueBeforeUpDown` [defaults to null](https://github.com/rickhanlonii/react-autosuggest/blob/e5336c6e890db37ceb5cd2a1f9eb60d33c2f210a/src/AutosuggestContainer.js#L102-L102) and is set to null in [a few places](https://github.com/rickhanlonii/react-autosuggest/blob/fde093db1a4630661f31cf1ae4d5f309b2c39de9/src/reducerAndActions.js#L106), which means that clients can receive `null` in `onChange` events. If they don't check for null and convert to the empty string (see the demos, which just set the `value` to what's passed), they can set `value` to null. 

This is what happens in https://github.com/moroshko/react-autosuggest/pull/165#discussion_r62436123.

So the issue here isn't that clients are passing `null` it's that they're passing through a `null` given to them by the library.